src/index.js

```diff
@@ -1,15 +1,32 @@
 var SentryCli = require('@sentry/cli');
 
+var SOURCEMAPS_OPTIONS = [
```

Comment:
@jan-auer i'm not sure if we shouldn't somehow expose this from the CLI. This way we won't have to update this list every time we add/remove an option from it.

Reply:
How about not having it in webpack-plugin at all? Just pass options through

Reply:
That makes sense as well. We can handle invalid options on CLI side.